src/main/webapp/app/exam/participate/exercises/modeling/modeling-exam-submission.component.ts

```diff
@@ -13,6 +13,8 @@ import { Exercise } from 'app/entities/exercise.model';
     templateUrl: './modeling-exam-submission.component.html',
     providers: [{ provide: ExamSubmissionComponent, useExisting: ModelingExamSubmissionComponent }],
     styleUrls: ['./modeling-exam-submission.component.scss'],
+    // change deactivation must be triggered manually
```

Comment:
You mean `change detection must be triggered manually` right? I am a Angular noob :D, so just a question, why is the default behaviour changed here only for the modeling-exam-submission component and not for the other ones like text and so on?